apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/nlog-jsonTemplate.xml

```diff
@@ -0,0 +1,72 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+  <extensions>
+    <add assembly="Google.Cloud.Logging.NLog" />
+  </extensions>
+  
+  <targets async="true">
+    <target type="GoogleStackdriver" projectId="PROJECT_ID" logId="LOG_ID" name="stackdriver" resourceType="container" sendJsonPayload="true" enableJsonLayout="true" includeCallSite ="true" includeCallSiteStackTrace="true" IncludeEventProperties="false" IncludeMdlc="true">
+      <contextproperty name="contextProp" layout="propContextLayout" />
+      <layout xsi:type="JsonLayout" includeAllProperties="true">
+        <attribute name="someProp" layout="prop" />
+        <attribute name="nestedProp" encode="false" >
+          <layout type='JsonLayout'>
+            <attribute name='insideNested1' layout='hello nested1!' />
+            <attribute name='insideNested2' layout='hello nested2!' />
+            <attribute name="nestedNestedProp" encode="false" >
```

Comment:
Please remove the space before the `>` in all of these elements. (It's fine to have one before the `/>` though.)